demos/chat_with_pdf.py

```diff
@@ -117,14 +130,16 @@ def initialize_components(self):
         # Initialize rerankers
         self.RERANKERS = {
             "HuggingFace": HFCrossEncoderReranker("BAAI/bge-reranker-base"),
-            "Cohere": CohereReranker(limit=3, api_config={"api_key": self.cohere_api_key})
+            "Cohere": CohereReranker(
+                limit=3, api_config={"api_key": self.cohere_api_key}
+            ),
         }
 
         self.openai_client = OpenAI(api_key=self.openai_api_key)
 
         # Initialize RAGAS evaluator chains
-        self.faithfulness_chain = EvaluatorChain(metric=faithfulness)
-        self.answer_rel_chain = EvaluatorChain(metric=answer_relevancy)
+        # self.faithfulness_chain = RagasEvaluatorChain(metric=faithfulness)
```

Comment:
this can now be deleted will do shortly

Reply:
Sounds good, any reason to go to gpt-3.5-turbo?